tools/rack/rack.plugin/plugin.xml

```diff
@@ -428,6 +438,27 @@
                 </visibleWhen>
             </command>
         </menuContribution>
+        <menuContribution label="zipIngestionPackage"
+            locationURI="popup:org.eclipse.ui.navigator.ProjectExplorer#PopupMenu?after=rackplugin.commands.uploadNodegroups">
+            <command
+                commandId="rackplugin.commands.zipIngestionPackage"
+                label="Save Ingestion Package as Zip" name="Zip Ingestion Package">
+                <visibleWhen>
+                    <with variable="selection">
+                        <count value="+" />
+                        <iterate ifEmpty="false" operator="and">
+                            <or>
+                                <instanceof
+                                    value="org.eclipse.core.resources.IProject" />
+                                <instanceof value="org.eclipse.core.resources.IFile" />
```

Comment:
@chrisage to test with multiple files selected from separate projects - need to add tests for the zipCreation method

Reply:
Multiple files are currently not supported. Added a separate issue. Also that the RackConsole is not hown when activate() is called.